.github/workflows/plugin-update.yml

```diff
@@ -52,13 +52,13 @@ jobs:
           git config user.name hc-github-team-secure-vault-ecosystem
           git config user.email hc-github-team-secure-vault-ecosystem@users.noreply.github.com
 
-      - if: steps.metadata.outputs.is-ent-branch != 'true'
+      - if: ! contains(inputs.branch, '+ent')
```

Comment:
If I am understanding correctly, this will only work if we always include the `+ent` version metadata in the plugin version. We don't currently do that for plugins that have always been Enterpise-only. For example https://github.com/hashicorp/vault-plugin-secrets-kmip/tags. However, if this is the approach you would recommend, then we can probably start doing that?

Reply:
No worries. I think I'll just pass in whether or not it's an enterprise plugin from the caller which can be derived from the plugins.yaml?